Cargo.toml

```diff
@@ -80,6 +81,7 @@ moved-trie = { path = "trie" }
 once_cell = "1.19"
 op-alloy = { version = "0.12", features = ["full", "std", "k256", "serde"] }
 openssl = "0.10"
+rand = "0.8"
```

Comment:
Just curious, there is no `[workspace.dev-dependencies]` right?

Reply:
Correct, there is not.